galleon-plugins/src/main/java/org/wildfly/galleon/plugin/config/WildFlyPackageTasksParser20.java

```diff
@@ -356,7 +356,7 @@ private void parseLineEndings(final XMLStreamReader reader, final WildFlyPackage
                             parseLineEnding(reader, builder, true);
                             break;
                         case UNIX:
-                            parseLineEnding(reader, builder, true);
+                            parseLineEnding(reader, builder, false);
```

Comment:
The CI has failed because the license header in this class needs to be updated. It's probably asking for 2019 as the current year. It's a bit annoying to keep it up-to-date, sorry about that.

Reply:
Ah. I knew it had to be something simple I was missing thanks!